platforms/jvm/plugins-java-base/src/main/java/org/gradle/api/plugins/JavaPluginExtension.java

```diff
@@ -92,6 +98,9 @@ public interface JavaPluginExtension {
      * The {@link FeatureSpec#capability(String, String, String)} method can be
      * used to refine the capabilities of this feature.
      *
+     * This method should not be called if there are multiple JVM {@link org.gradle.api.component.Component}s
```

Comment:
Like we talked about, I don't think we should mention components yet

Reply:
Seems like this is still unresolved